components/ide/ide-ui-generate-service/src/main/resources/META-INF/dirigible/ide-generate-service/template/parameterUtils.js

```diff
@@ -162,6 +162,28 @@ export function process(model, parameters) {
             parameters.perspectives[e.perspectiveName].views.push(e.name);
         }
     });
+
+    parameters.roles = [];
+
+    model.entities.forEach(e => {
+        if (e.generateDefaultRoles === "true") {
+            if (e.type != "PROJECTION") {
+
+                let rolePair = {};
```

Comment:
In JavaScript always use `const` instead of `let`, unless there is reassignment of the variable at a later point.